src/wallet/wallet.cpp

```diff
@@ -4145,3 +4148,14 @@ bool CMerkleTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState&
 {
     return ::AcceptToMemoryPool(mempool, state, tx, true, NULL, NULL, false, nAbsurdFee);
 }
+
+bool CalculateEstimateType(FeeEstimateMode mode) {
```

Comment:
If you change the semantics of the return value all the other callers will have to change too. Why not just have the caller consume the returned mode? Anyways, not a blocker.